### PR TITLE
Using Cupertino Widgets

### PR DESCRIPTION
**Pull request by MapleSyyrup**
- IOS has its own design of widgets called Cupertino.
- Add adaptive widgets for IOS platform

Note: The chart is below the appbar in IOS platform. This will be fixed on the next update.
image

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -133,58 +146,66 @@ class _MyHomePageState extends State<MyHomePage> {
...
-      floatingActionButtonLocation: FloatingActionButtonLocation.centerFloat,
-      floatingActionButton: Platform.isIOS
-          ? Container()
-          : FloatingActionButton(
-              child: Icon(Icons.add),
-              onPressed: () => _startAddNewTransaction(context),
+
+    final pageBody = SingleChildScrollView(
+      child: Column(
+        // mainAxisAlignment: MainAxisAlignment.start,
```
Comment (Nei1eveN) [inline lib/main.dart]: do you still need this? if not, please remove this line

File: lib/main.dart [hunk 2]
```diff
@@ -133,58 +146,66 @@ class _MyHomePageState extends State<MyHomePage> {
...
+          if (!isLandscape)
+            Container(
+              height: (mediaQuery.size.height -
+                      appBar.preferredSize.height -
+                      mediaQuery.padding.top) *
+                  0.3,
+              child: Chart(_recentTransactions),
+            ),
+          if (!isLandscape) txListWidget,
+          if (isLandscape)
```
Comment (Nei1eveN) [inline lib/main.dart:179]: isn't this supposed to be if else?

`if (isLandscape) _showChart ? Container() : txListWidget`

Comment (Nei1eveN) [reply lib/main.dart:179]: or can you try

```
if (isLandcape) {
...[
Widgets1,
Widgets2,
]
} else {
...[
Widgets1,
Widgets2,
]
}
```

Comment (MapleSyyrup) [reply lib/main.dart:179]: if else is other way of doing it. using only if is fine.

Comment (MapleSyyrup) [reply lib/main.dart:179]: This is a special 'if inside a list' syntax from the new update and curly braces can't be used.

**Review by Nei1eveN — APPROVED** [review]

**Pull request merged**